### PR TITLE
Add stream name to buffer overflow log message

### PR DESCRIPTION
**Pull request by sad-fat-cat**
Refs #28085

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @enp0s23,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by raboof** [comment]
OK TO TEST

**Review by raboof — COMMENTED** [review]
I like adding this information.

Perhaps instead of adding a prefix we could add it as a postfix `in stream [{}]`? That might make it clearer that while there might be a failure in the stream, that does not necessarily mean the whole stream is failing (you might have a recover somewhere).

**Comment by raboof** [comment]
(it looks like a line got so long scalafmt doesn't like it anymore - perhaps you could run `scalafmtAll` to have it fixed?)

File: akka-stream/src/main/scala/akka/stream/impl/ActorRefSource.scala [hunk 1]
```diff
@@ -71,17 +72,18 @@ private object ActorRefSource {
           buffer match {
             case OptionVal.None =>
               if (isCompleting) {
-                log.warning("Dropping element because Status.Success received already: [{}]", m)
+                log.warning("[{}] Dropping element because Status.Success received already: [{}]", name, m)
```
Comment (hepin1989) [inline akka-stream/src/main/scala/akka/stream/impl/ActorRefSource.scala]: dropping

**Comment by raboof** [comment]
@enp0s23 this looks pretty close to being mergable, do you want to do the last bit or could someone else take over?

**Review by johanandren — APPROVED** [review]
Not entirely sure that it makes sense to use the class name over to string of the logic, but I think we can start out with this and see if anyone complain.

**Review by raboof — APPROVED** [review]

**Pull request merged**